repository-data/application/src/main/resources/hcm-config/configuration/translations/templates.yaml

```diff
@@ -24,14 +25,19 @@ definitions:
       new-supplementary-info-folder: new folder
       new-supplementary-info-hub-document: new supplementary information hub document
       new-task-document: new task document
-      new-task-folder: new folder
+      new-task-folder: new task folder
```

Comment:
Weren't you saying you're going to remove this?

Reply:
As in, do we need special queries (and translations) for `new-task-document` and `new-task-folder` ?